### PR TITLE
Windows client imagery

### PR DESCRIPTION
**Pull request by mdescalzo**
Added window 10 imagery.

### CODE REVIEW

**Comment by mayfield** [comment]
can we put those assets in electron or /resources?  Otherwise they get included in the web servers' static folder and bloat the heroku slug too.

**Comment by mdescalzo** [comment]
I've moved the assets to electron/windowsAssets and modified the scripts accordingly.

File: electron/output/Forsta Messenger/pre-appx/app/LICENSE [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2013-2018 GitHub Inc.
```
Comment (mayfield) [inline electron/output/Forsta Messenger/pre-appx/app/LICENSE]: copyright github?

Comment (mdescalzo) [reply electron/output/Forsta Messenger/pre-appx/app/LICENSE]: Weird.  I'll dig into where that came from.

**Comment by mayfield** [comment]
I just noticed this has hundreds of binaries in it.   We may need to scratch these commits alltogether to keep them out of the git repo thus causing permanent project bloat.

**Comment by mdescalzo** [comment]
Agreed.  I just caught that myself.

**Pull request closed without merging**